__tests__/app.test.js

```diff
@@ -71,4 +71,18 @@ describe("GET: /api/articles/:article_id", () => {
             expect(response.body.msg).toBe('Invalid ID');
         })
     })
+})
+
+describe('/api/articles', () => {
+    test('GET:200 sends an array of article objects with the correct keys', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+          expect(response.body.articles).toEqual(expect.any(Array));
+          expect(Object.keys(response.body.articles[0])).toEqual(
+            expect.arrayContaining(['author', 'title', 'article_id', 'topic', 'created_at', 'votes', 'article_img_url', 'comment_count'])
+          );
+        });
+    });
 })
```

Comment:
The ticket 5 instructions state:

`In addition:

the articles should be sorted by date in descending order.
there should not be a body property present on any of the article objects.`

So there are 2 more tests to do:
* to check the articles are in created_at order
* to check the articles are in descending order

jest-sorted will help with this.